.travis.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     if: tag IS present
     script: |
       docker login -u="${QUAY_BOT}" -p="${QUAY_PASSWORD}" quay.io
-      make release IMG_TAG="${TRAVIS_TAG}"
+      make release IMG_TAG="${TRAVIS_TAG}" VERSION="${TRAVIS_TAG}"
```

Comment:
You should trim the leading `v` from `TRAVIS_TAG`, because our release tags are `vx.y.z`

Reply:
Sorry, show be fixed now.
Note: I used `sed` instead of the bashism `${TRAVIS_TAG:1}`, not sure if travis-ci still runs `dash` instead of `bash` but just in case...